doc/reports/spikes/metrics-what-we-want-spike.md

```diff
@@ -0,0 +1,18 @@
+# Monitoring Performance - What should we monitor?
+## Introduction
+This spike report follows on from work and discussions in the OAT meeting to identify areas where we want to be monitoring the system. This outlines what I see as an idealised monitoring setup - there may be blockers to overcome to achieve some of this. I've also provided guidance in the document to how I see the monitoring being provided and used.
+## Production metrics
+All of these metrics should be radiated on monitors
+### Performance
+* **99% call time** - This can be pulled from Google Analytics, CloudWatch or Zipkin headers in Splunk logs, potentially further broken down by microservice
```

Comment:
Might be a stupid question but what does this bullet point mean?

Reply:
99% call time is the time below which 99% of page calls are served up. Our self-imposed non-functional requirement is that this is below 2 seconds :)

Reply:
So it's a 99% success rate?

Actually seems rather low, tbh, but it's a start!

Reply:
Are we due NFRs from anyone?  Blue sky NFRs are often not useful.

Reply:
Na, not to be confused with 99(.9999999)% uptime. This is simply the time that 99% of users can expect to get, as opposed to the median (or 50% call time)